pkg/scheduler/server/internal/etcd/etcd.go

```diff
@@ -135,3 +144,48 @@ func (e *etcd) Close() error {
 
 	return err
 }
+
+func (e *etcd) maybeDeleteDataDir() error {
+	_, err := os.Stat(e.existingClusterPath)
+	if err == nil {
+		log.Infof("Found existing cluster data flag, preserving data dir: %s", e.config.Dir)
+		return nil
+	}
+
+	if !os.IsNotExist(err) {
+		return err
+	}
+
+	log.Infof("No existing cluster data flag found, deleting data dir contents: %s", e.config.Dir)
```

Comment:
I dont think that'll mean anything to a user so I think we can just rm that key work.
```suggestion
	log.Infof("No existing cluster data found, deleting data dir contents: %s", e.config.Dir)
```